### PR TITLE
Fix for es6 import error

### PR DESCRIPTION
**Pull request by smilledge**
Fix for the import error when using es6 import syntax.

https://github.com/bunkat/later/issues/153

### CODE REVIEW

**Comment by icfantv** [comment]
@bunkat can we please get this merged and released?  it's blocking our upgrade to webpack 2.  Otherwise, we'll have to drop the library and find a different solution.

**Comment by Chexpir** [comment]
@bunkat can we please get this merged and released? We are using webpack2 and ussing a forked version in the meantime :-)

**Comment by icfantv** [comment]
@Chexpir, due to the lack of response and this project being MIT licensed, we simply pulled it in-house.  Not ideal, but better than continually waiting.

File: index.js [hunk 1]
```diff
@@ -5,11 +5,15 @@ globals.forEach(function(g) {
   if (g in global) globalValues[g] = global[g];
 });
 
-require(process.env['LATER_COV'] ? "./later-cov" : "./later");
+if (process.env['LATER_COV']) {
```
Comment (seveves) [inline index.js]: you should also check if process is defined before trying to access env. webpack will throw an error here. I did the same fix in #208

**Comment by icfantv** [comment]
Given that it's been six months since this PR was opened and the last post by the author of this project was in October of '16 requesting help since he no longer used it, I think it safe to assume this is abandoned.  If anyone wants to fork it and correct the issues, that's probably the best course of action at this point.

**Comment by gabrielgrant** [comment]
@icfantv If someone is willing to do maintenance work, it would probably be better for them to just reply to the [lib author's call for help](https://github.com/bunkat/later/issues/184) and make the fixes here, no? Unfortunately afaics no one has stepped up to do so yet

**Comment by kirkins** [comment]
@icfantv hi wondering if you are still maintaining an in-house version of this library? Care to make it public if so?

**Comment by icfantv** [comment]
@kirkins I wouldn't say _maintaining_.  Rather, we pulled it in house and fixed the bug.  I don't recall it being touched since.  Looks like it was also moved so we lost the history.  Here's the current version:  https://github.com/spinnaker/deck/tree/master/app/scripts/modules/core/src/utils/later

**Comment by icfantv** [comment]
@kirkins Sure.

**Comment by niftylettuce** [comment]
This issue is now fixed per my work with **[Bree](https://github.com/breejs/bree)**, I have an updated fork of this package called `@breejs/later`.  See https://github.com/breejs/later if you have additional PR's or issues to file.  Contributions welcome to clean it up.

**Comment by smilledge** [comment]
Closing as https://github.com/bunkat/later/pull/208 is better

**Pull request closed without merging**